src/Dynamo.ts

```diff
@@ -25,8 +26,18 @@ import {
 
 import {marshall, unmarshall} from '@aws-sdk/util-dynamodb'
 
+export interface DynamoParams {
+    client: DynamoDBClient,
+}
+
 export class Dynamo {
-    constructor(params = {}) {
+    client: DynamoDBClient // TODO Confirm type
```

Comment:
Is this the right type? I wasn't entirely sure what this wrapper converts from/to

Reply:
That looks right:

import {DynamoDBClient} from '@aws-sdk/client-dynamodb'
const client = new DynamoDBClient(params)